mac/Keyman4MacIM/Keyman4MacIM/en.lproj/Localizable.strings

```diff
@@ -68,4 +68,7 @@
 /* copyright label in the Package Information window */
 "copyright-label" = "Copyright:";
 
+/* message displayed to alert user to need grant accessibility permission */
+"privacy-alert-text" = "To function properly, Keyman requires accessibility features.\n\nAfter granting access in System Preferences, Security & Privacy, restart your system.";
```

Comment:
Does the user really have to restart macOS in order for things to run correctly? Is there a pathway we can use to restart the input method without having to do a reboot?

Reply:
Okay, future enhancement I guess!